### DISCUSSION TITLE
MAX_LIST_LENGTH

### DISCUSSION THREAD

**Discussion by cyclicscooby** [sstaub/NextionX2#3] [General]
Firstly, thanks for the lib... 
I've reached a limit for touch components on a page.. 
I looked thru the lib and found 'MAX_LIST_LENGTH 24', which is the limit i'm hitting..
Is it this length for a reason..? Something to do with the buffer length..?

I'm running on a Teensy4 and need a max of 40 touch objects.. 

Thanks

**Comment by sstaub** [comment]
For use with Teensy you can use more than 24 objects. Because Arduino has less RAM there is this limit to 24 objects.

**Comment by cyclicscooby** [comment]
Thankyou sstaub.. Much appreciated.. x
